components/access_problem/index.tsx

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useEffect} from 'react';
+import {useIntl} from 'react-intl';
+
+import {trackEvent} from 'actions/telemetry_actions';
+
+import AccessProblemSVG from 'components/common/svg_images_components/access_problem_svg';
+
+import './access_problem.scss';
+
+const AccessProblem = () => {
+    const {formatMessage} = useIntl();
+    useEffect(() => {
+        trackEvent('signup', 'click_login_no_account__closed_server');
+    }, []);
+    return (
+        <div className='AccessProblem'>
```

Comment:
I think the outer div.AccessProblem and child div.AccessProblem__container are not necessary, you would only probably need the div.AccessProblem__body to work as a wrapper and that is it